rest/blip_sync.go

```diff
@@ -978,12 +978,23 @@ func (ctx *blipSyncContext) setActiveSubChanges(changesActive bool) {
 
 // setUseDeltas will set useDeltas on the blipSyncContext as long as both sides of the connection have it enabled.
 func (ctx *blipSyncContext) setUseDeltas(clientCanUseDeltas bool) {
+	// If either side don't want deltas, log and exit early
+	if !ctx.sgCanUseDeltas || !clientCanUseDeltas {
+		ctx.Logf(base.LevelInfo, base.KeySync, "Disabling deltas for this replication. SG: %t - Client: %t", ctx.sgCanUseDeltas, clientCanUseDeltas)
```

Comment:
```suggestion
                 if sgCanUseDeltas {
		   ctx.Logf(base.LevelInfo, base.KeySync, "Disabling deltas for this replication based on client setting.")
		} 
```
I think we only need to log at info level here if SG has deltas enabled, but the client replication is disabling that.  The question of 'are deltas enabled for SG' is already answered w/ startup logging.